common/validations.js

```diff
@@ -193,3 +193,9 @@ export const isUnityFile = async (file) => {
     return false;
   }
 };
+
+export const isUsernameMatch = (source, candidate) => {
```

Comment:
I'm usually all for breaking down things into more manageable functions, but I think this is one case where it's unnecessary and just adds more code and confusion. This method is essentially just `a === b` and should be left as that in the code rather than making this extra function which people would have to track down to see what it is

Reply:
okay sounds good, ill just move it to the component